datasets/hebrew_projectbenyehuda/hebrew_projectbenyehuda.py

```diff
@@ -0,0 +1,141 @@
+"""Public domain texts from Project Ben-Yehuda- a set of books extracted from the Project BenYehuda library"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@article{,
+  author = {},
+  title = {Public domain texts from Project Ben-Yehuda},
+  journal = {},
+  url = {https://github.com/projectbenyehuda/public_domain_dump},
+  year = {2020},
+}
+
+"""
+
+_DESCRIPTION = """\
+This repository contains a dump of thousands of public domain works in Hebrew, from Project Ben-Yehuda, in plaintext UTF-8 files, with and without diacritics (nikkud). The metadata (pseudocatalogue.csv) file is a list of titles, authors, genres, and file paths, to help you process the dump.
+All these works are in the public domain, so you are free to make any use of them, and do not need to ask for permission.
+There are 10078 files, 3181136 lines
+"""
+
+_ASSET_ROOT_URL = "https://raw.githubusercontent.com/projectbenyehuda/public_domain_dump/master/"
+_STORAGE_API_ROOT_URL = "https://raw.githubusercontent.com/projectbenyehuda/public_domain_dump/master/txt/"
+
+# download one by one file from github is too slow
+
+_METADATA_URL = _ASSET_ROOT_URL + "pseudocatalogue.csv"
+
+
+class HebrewProjectBenYehuda(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class HebrewProjectbenyehuda(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `hebrew_projectbenyehuda`

Reply:
Hi, I fixed it, but got some strange errors from CI, like "../.local/lib/python3.6/site-packages/xlrd/__init__.py:170: XLRDError" I didn't use excel at all, probably it relates to other datasets